lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -497,4 +498,17 @@ public S3FileSystemConfig setNonProxyHosts(String nonProxyHosts)
         this.nonProxyHosts = ImmutableSet.copyOf(Splitter.on(',').omitEmptyStrings().trimResults().split(nullToEmpty(nonProxyHosts)));
         return this;
     }
+
+    public boolean isSupportsExclusiveCreate()
+    {
+        return supportsExclusiveCreate;
+    }
+
+    @Config("s3.exclusive-create")
+    @ConfigDescription("Whether S3-compatible storage supports exclusive create (true for Minio and AWS S3)")
```

Comment:
does it also warrant some documentation in file-system-s3.md?

Reply:
Added